Grid/grid.css

```diff
@@ -0,0 +1,33 @@
+.grid-container{
+    width: 90%;
+    display: grid;
+    padding: 2em;
+    grid-gap: 10px;
+    margin: 1rem;
+    border: 1px solid rgb(160, 158, 158);
+}
+.two-col-grid{
+    grid-template-columns: 1fr 1fr;
+    
+}
+
+.three-col-grid{
+    grid-template-columns: 1fr 1fr 1fr;
+}
+
+.grid-item{
+    background-color: #84c6fc;
```

Comment:
you should use CSS variable as it is a best practice to follow

Reply:
since this is not repeating css var is not used here